cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release_strategy.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: push-to-external-registry-strategy
+spec:
+  params:
+    - name: extraConfigGitUrl
+      value: https://github.com/hacbs-release/strategy-configs.git
```

Comment:
do you have a PR for this?

Reply:
not yet

Reply:
https://github.com/hacbs-release/strategy-configs/pull/17